app/src/main/java/com/strv/movies/ui/loading/LoadingScreen.kt

```diff
@@ -0,0 +1,36 @@
+package com.strv.movies.ui.loading
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import com.strv.movies.ui.theme.MoviesTheme
+
+@Composable
+fun LoadingScreen(
+    modifier: Modifier = Modifier
+) {
+    Column(
```

Comment:
`ErrorScreen` is `Box`, is there any reason for having `Column` here?

Reply:
I'm just showing that it can be done another way.